### PR TITLE
RPC update: `getGov`

### PR DESCRIPTION
**Pull request by ivan-zynesis**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Data structure change in RPC response

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
Part of #532 
Changes include #789 (require new setGov rpc)

### CODE REVIEW

**Review by canonbrother — APPROVED** [review]

File: packages/jellyfish-api-core/src/category/masternode.ts [hunk 1]
```diff
@@ -141,15 +141,49 @@ export class Masternode {
...
+   * @param {string} name governance name
+   * @return {Promise<Array<Record<string, any>>} governance information as json object
+   * @example
+   * {
+   *   "LP_SPLITS": { "3": BigNumber, "4": BigNumber },
+   *   "5000": { "3": BigNumber, "4": BigNumber }, // value after block height 5000
+   *   "8000": { "3": BigNumber, "4": BigNumber } // value after block height 8000
+   * }
+   */
+  async getGovWithHeight (name: string): Promise<Array<Record<string, any>>> {
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/src/category/masternode.ts:183]: Due to regression in https://github.com/DeFiCh/jellyfish/pull/753 we can't deprecate getGov, we have to replace it and fix all the tests that broke due to it. This change has to be moved to #753 due to various regression

Comment (ivan-zynesis) [reply packages/jellyfish-api-core/src/category/masternode.ts:183]: Porting changes by hand, no more multiple versions.

Comment (surangap) [reply packages/jellyfish-api-core/src/category/masternode.ts:183]: was checking why we need `Array<Record<string, any>>` in the return object. This is the structure of the return data
```
 [
      { LP_SPLITS: { '3': [BigNumber], '4': [BigNumber] } },
      { '109': { '3': [BigNumber], '4': [BigNumber] } },
      { '114': { '3': [BigNumber], '4': [BigNumber] } }
    ]
```

seems bit odd to me. Why can't  it return `Record<string, any>` type. i.e
```
{
      LP_SPLITS: { '3': [BigNumber], '4': [BigNumber] },
      '109': { '3': [BigNumber], '4': [BigNumber] } ,
      '114': { '3': [BigNumber], '4': [BigNumber] }
}
```
does it need the extra Array?

Comment (surangap) [reply packages/jellyfish-api-core/src/category/masternode.ts:183]: btw we also have a `listgovs` RPC

**Comment by ivan-zynesis** [comment]
Defid reverted

**Pull request closed without merging**